website_sale_product_detail_attribute_image/static/tests/tours/website_sale_product_detail_attribute_image.js

```diff
@@ -22,7 +27,7 @@ odoo.define("website_sale_product_detail_attribute_image.tour", function (requir
         {
             url: "/shop",
             test: true,
-            wait_for: base.ready(),
+            wait_for: ready,
```

Comment:
Is it needed at all?

Reply:
It is not necessary, in 15.0 it was configured with the `wait_for` parameter and I migrated it as a precaution, I am going to delete it.
Sorry for the delay.

Reply:
Done